examples/demo/site/modules/auth/mail_templates/account_evaluation.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
```

Comment:
It would be good to qualify the templates 
and prefix with "admin" when it will be used to send email to admin.